cdc/scheduler/internal/v3/replication/replication_set.go

```diff
@@ -917,3 +917,46 @@ func (r *ReplicationSet) updateCheckpointAndStats(
 	}
 	r.Stats = stats
 }
+
+// ReplicationSetHeap is a max-heap, it implements heap.Interface.
```

Comment:
I think it should be a min heap, because we need to get the slowest checkpoint, aka, the minimum checkpoint.

Reply:
We use a max heap to get the k slowest tables, when a new table is `Push` into the heap, the top element of the heap, which is the fastest table in the heap will be `Pop` out. Eventually, the k slowest tables will remain in the heap.

Reply:
The fastest table will be `Pop` out in every iteration, finally the slowest tables remain.